src/matrix/matrixfdist.jl

```diff
@@ -0,0 +1,142 @@
+"""
+```julia
```

Comment:
Don't the docstrings usually start with the name of the distribution at the top? Might want to check others for consistency.

Reply:
Yes. A couple don't, but most do. The first thing in the `julia` block is the name, but the effect is the same if it's above it.